Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -1095,20 +1361,20 @@ def input_validation(command_name: str) -> None:
     Raises:
         DemistoException: Will be raised if input failed the validation
     """
-
+    command_data = COMMANDS_ARGS_DATA[command_name]
     if limit := inner_args_values.get('limit'):
         try:
             if int(limit) < 1:
                 raise ValueError()
         except ValueError as exc:
             raise DemistoException('Limit parameter must be an integer bigger than 0') from exc
-    if required_groups := COMMANDS_ARGS_DATA[command_name].get('required_groups'):
-        for group in required_groups:
-            for arg in group:
-                if arg in inner_args_values or arg in args_values:
-                    break
-            else:
-                raise DemistoException(f'At least one of the arguments {group} must be provided')
+
+    for group in command_data.get('required_groups', []):
+        existing_args = [arg for arg in group if arg in inner_args_values or arg in args_values]
+        if len(existing_args) != 1:
+            raise DemistoException(f'Exactly least one of the arguments {group} must be provided')
```

Comment:
please separate this for loop to another function and test it

Reply:
done